docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/01 - Identifiers.md

```diff
@@ -0,0 +1,75 @@
+# Identifiers
+
+## Overview
+By default LORIS provides 2 different identifiers for each participant or subject: 
+
+- **CandID**, also known as the **DCCID**, is a unique randomized 6-digit numeric ID (e.g. '436792') assigned automatically by LORIS upon participant registration. The CandID is not configurable.
+- **PSCID** (Project Study Center ID) is a unique configurable ID. It can be be setup to be either manually entered when registering a participant or automatically generated and usually contains the site abbreviation followed by sequential or randomized characters (e.g. 'MTL0006'), but its exact format is customizable.
+
+The format and the generation of the **PSCID** must be configured by an admin at the setup stage.
+
+## Configuration
+
+### Configuration from the front end
+_not yet available_
+
+### Configuration from SQL
+_not yet available_
+
+### Configuration from the config.xml file
+
+Both the format and the generation of PSCIDs can be configured by the project in the `config.xml` file of the `%LORIS_ROOT%/project/` directory.
+
+PSCIDs can be created for new subjects in one of 3 ways: *sequentially generated*, *manually entered*, or *randomly generated*.
+
+1. ***sequential*** generates PSCIDs sequentially for each new participant registered. **(default)**
+
+ ```xml
+ <PSCID>
+     <generation>sequential</generation> 
+     <structure>
+         <seq type="siteAbbrev"/>
+         <seq type="numeric" minLength="2" maxLength="4" min="10" max="9999"/>
+     </structure>
+ </PSCID>
+ ```
+
+2. ***manual*** asks the user to enter the PSCID when registering a new participant.
+
+ ```xml
+ <PSCID> 
+     <generation>user</generation> 
+     <structure>
+         <seq type="alphanumeric" minLength="2"/> <!-- Ex: AAA1-->
+     </structure>
+ </PSCID>
+ ```
+
+3. ***random*** generates PSCIDs with a random numerical value for each new participant registered.
+
+ ```xml
+ <PSCID>
+     <generation>random</generation> 
+     <structure>
+         <seq type="static">PROJ</seq>
+         <seq type="siteAbbrev"/>
+         <seq type="numeric" minLength="4" maxLength="4" min="0001" max="9999" />
+     </structure>
+ </PSCID>
+ ```
+ 
+ Options for the `type` element of the `<seq>` tag are:
+  - `siteAbbrev`: A string value that will be used as a dynamic prefix. Value drawn from the `Alias` field of the `psc` table in the database.
+  - `static`: A string value that will be used as a fixed prefix. Value defined in the `config.xml` file.
+  - `numeric`: An integer value generated dynamically in accordance to the generation method defined.
+  - `alphanumeric`: An alphanumeric string value generated dynamically in accordance to the generation method defined. 
+  - `alpha`: An alphabetic string value generated dynamically in accordance to the generation method defined.
+ 
+  > Note: The types above can be associated with a `minLength` and/or a `maxLength` attribute. both `minLength` and `maxLength` default to `4` if not specified.
```

Comment:
I think it would be better to specify in some way that the note apply to `numeric`, `alphanumeric` and `alpha` types only.  Something like ` The last 3 types above can ...`

Reply:
done